project/pd/project.json

```diff
@@ -0,0 +1,31 @@
+{
+    "maintainer": [
```

Comment:
Given that PD and TiKV can't be used alone and PD committers are also actively updating TiKV, I suggest to merge pd project and tikv project.

/cc @nolouch

Reply:
There are many PD contributors that are not updating TiKV and PD can run with unistore or TiFlash (after it supports direct write) and there are proposals not TiKV related.

Reply:
> There are many PD contributors that are not updating TiKV

You can find that all PD maintainers, committers have updated TiKV except qiuyesuifeng.

> PD can run with unistore or TiFlash (after it supports direct write)

Unfortunately, unistore is not expected to be used in production, and it's purpose is to provide better optimization direction for TiKV. TiFlash is using TiKV to communicate with PD.

The main goal is to have a role that can decide from the global view and choose the design that best suit the evolution of product.

Reply:
TiFlash is mainly using the Raft layer of TiKV but not the whole TiKV.

IMO, PD provides the ability to schedule data partition (called `Region`) under multi-raft, which is a separate layer from TIKV, a specific key-value storage implementation.

Reply:
> TiFlash is mainly using the Raft layer of TiKV but not the whole TiKV.

This doesn't change the fact that PD is talking to TiKV instead of Tiflash.

> IMO, PD provides the ability to schedule data partition (called Region) under multi-raft...

It's not true. PD's behavior is tightly bound to the behavior of TiKV. Of cause you can simulate a TiKV using the same protocols, but its design is not for general KV storage.

Reply:
@BusyJay @Jayson-Huang @disksing Thanks for your comments. The main issue here might not be software related, which we will discuss on RoadMap and even all its history and future.

The main issue here is about governance. Is PD as a separated project able to create their own proposal and finish development? The original argument I learn from @BusyJay  is that all of PD RFC is coupled with TiKV and most of TiKV proposals contains a "What to do on PD" part.

If we can clarify that

* PD can propose its own (and only) proposal.
* A cross repo proposal can be divided into two or more proposals that can be located in repositories in one project.

Then it should not be a bad direction PD grows its own community.

If current PD members, as individuals, also works on TiKV project repositories, there is no problem those actively contribute to TiKV also reviewer / committer on TiKV project.

Reply:
@BusyJay @nolouch It seems PD members believe the project can grow its own community.

@nolouch do you agree that if there is a cross project proposal, said involving TiKV and PD, you'd help it to be divided into two or more proposals each of which can be located in repositories in one project? I think it is the original bad case proposed, that when you create an RFC on PD, it must associated with TiKV and thus @BusyJay supposes they share the same membership.

@BusyJay do you have further concern?

Reply:
ping @BusyJay @nolouch

Reply:
I think the proposal does not necessarily require separation, I would like to make one RFC for one feature, I think put them in one org is accepted. such as `tikv/rfcs` already has [0013-schedule-limit.md](https://github.com/tikv/rfcs/blob/master/text/0013-schedule-limit.md) it only relative PD, and  [0054-joint-consensus.md](https://github.com/tikv/rfcs/blob/master/text/0054-joint-consensus.md) relative two project.

Reply:
@nolouch I think I get your point. We can label an RFC with related projects and require at lease X (maintainer|committer) approval of each project to get the RFC merged.

Reply:
Ping @BusyJay we may think it as consensus reached to leave TiKV & PD team as is if there is no more objection. We can discuss later on the description how to accept an RFC. Roughly we tag RFC with team label and require X committers of each team give LGTM.

Reply:
LGTM